docs/resources/samples/combine-worksheets-into-single-workbook.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Combine workbooks into a single workbook'
+description: 'Learn how to use Office Scripts and Power Automate to create merge worksheets from other workbooks into a single workbook.'
+ms.date: 09/03/2021
+localization_priority: Normal
+---
+
+# Combine worksheets into a single workbook
+
+This sample shows how to pull data from multiple workbooks into a single, centralized workbook. It uses two scripts: one to retrieve information from a workbook and another to create new worksheets with that information. It combines the scripts in a Power Automate flow that acts on an entire OneDrive folder.
+
+> [!IMPORTANT]
+> This sample only copies the values from the other workbooks. It does not preserve formatting, charts, tables, or other objects.
+
+## Scenario
+
+1. Create a new Excel file in your OneDrive and add two scripts from this sample to it.
+1. Create a folder in your OneDrive and add at least one workbook with data to it.
+1. The flow you'll build gets all the files in a specified folder full of Excel workbooks.
+1. The **Return Worksheet Data** script gets the data from every worksheet in each of the workbooks.
```

Comment:
The script names should be consistent in capitalization, and I recommend only capitalizing the first letter of first word per style guide. Other Office Scripts sample scripts seem to be using only first letter of first word.
```suggestion
1. The **Return worksheet data** script gets the data from every worksheet in each of the workbooks.
```